thrusterCalibration/runThrusters.ino

```diff
@@ -45,26 +54,59 @@ int forwardValue3 = 1550;
 int backwardValue3 = 1400;
 
 char buffer[512];
-int killSwitch = 0;
 
+int killSwitch = 0;
+//this pin is for toggling on/off the robot
 int starterPin = 28;
+
+void setup() {
+  //Sets USB Baud Rate - Serial Monitor must match
+  /*
+  Baud rate = rate in bits per second for communications
+  */
+  Serial.begin(9600);
+  //Set up the starter pin to read input from a signal
+  pinMode(starterPin, INPUT);
+  //while robot not enaged wait for commad after staterpin is turned on
+  while( digitalRead(starterPin) != HIGH ) {
+    Serial.println("waiting");
+  }
+  //after enaged tell computer over serial it is ready and set kill switch to waiting mode
+  Serial.println("starting..");
+  killSwitch = 1;
+  //wait 5 seconds
+  delay(5000);
+  //prepare the servos
+  setupServo(servo2, 2);
+  setupServo(servo3, 3);
+  setupServo(servo4, 4);
+  setupServo(servo5, 5);
+//  setupServo(servo6, 6);
+//  setupServo(servo7, 7);
+  Serial3.begin(9600);
+  Serial.begin(9600);
+  delay(2000);
+  
+}
   
 void loop() {
+  //check that starter is not off
```

Comment:
these two if statements were just a way around how the robot was wired. It's some recovery logic that just needs to go away.

I'm probably wrong, but I think it was due to recover from a subsystem randomly losing power. Anyways we're probably better off if they're just sent off to the bitbucket in the sky.